transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -24,14 +24,18 @@ class TrackStatusMessage {
 export interface SmallWebRTCTransportConstructorOptions {
   iceServers?: RTCIceServer[];
   waitForICEGathering?: boolean;
+  /** @deprecated Use webrtc_url instead */
   connectionUrl?: string;
+  webrtc_url?: string;
```

Comment:
Hmm, this ought to be camelCase, since we're in JS here (and all the other options next to it are camelCase). As is, this kinda sticks out like a sore thumb.

I'm guessing the reason for snake_case here is because the equivalent options in the other transports already are snake_case? If so...I might lobby for updating them across the board rather than doubling down on this pattern.

(I know this feels annoyingly nitpicky but IMO this kind of public API polish does make stuff feel nicer to use)

Reply:
it's more than that. It's because these values are often sent to us from a server endpoint which has generally send responses snake cased :/

So it's more about what we want to standardize in those payloads to make `client.startBot().then((response) => client.connect(response))` just work.

might be worth a quick chat??

Reply:
also related: https://pluot.slack.com/archives/C07V2LX3GER/p1754308816020919